packages/astro/src/i18n/utils.ts

```diff
@@ -232,6 +233,32 @@ export function toRoutingStrategy(
 	return strategy;
 }
 
+export function fromRoutingStrategy(
+	strategy: RoutingStrategies,
+	fallbackType: NonNullable<SSRManifest['i18n']>['fallbackType'],
+): NonNullable<AstroConfig['i18n']>['routing'] {
+	let routing: NonNullable<AstroConfig['i18n']>['routing'];
+	if (strategy === 'manual') {
+		routing = 'manual';
+	} else {
+		routing = {
+			prefixDefaultLocale:
+				strategy === 'pathname-prefix-always' ||
+				strategy === 'domains-prefix-always' ||
+				strategy === 'pathname-prefix-always-no-redirect' ||
+				strategy === 'domains-prefix-always-no-redirect',
+
+			redirectToDefaultLocale: !(
+				strategy == 'pathname-prefix-always-no-redirect' ||
```

Comment:
```suggestion
				strategy === 'pathname-prefix-always-no-redirect' ||
```

Typo. Might want to consider using a set or soemthing for both of these though

Reply:
To use a set how?

Reply:
Like have a `Set` of prefix strategies and a Set of no-redirect strategies, and instead of comparing the strategy with each one, one by one, do a set `has(strategy)`